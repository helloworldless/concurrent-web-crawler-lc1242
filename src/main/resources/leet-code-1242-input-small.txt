["http://news.yahoo.com","http://news.yahoo.com/news","http://news.yahoo.com/news/topics/","http://news.google.com","http://news.yahoo.com/us"]
[[2,0],[2,1],[3,2],[3,1],[0,4]]
"http://news.yahoo.com/news/topics/"
