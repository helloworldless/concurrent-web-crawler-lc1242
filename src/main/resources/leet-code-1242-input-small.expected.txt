["http://news.yahoo.com/news/topics/","http://news.yahoo.com/news","http://news.yahoo.com/us","http://news.yahoo.com"]